release/ray_release/byod/build.py

```diff
@@ -12,8 +12,8 @@
 from ray_release.test import Test
 
 DATAPLANE_S3_BUCKET = "ray-release-automation-results"
-DATAPLANE_FILENAME = "dataplane.tgz"
-DATAPLANE_DIGEST = "f9b0055085690ddad2faa804bb6b38addbcf345b9166f2204928a7ece1c8a39b"
+DATAPLANE_FILENAME = "dataplane_v2.tgz"
```

Comment:
could you change this to a date? like `20230622`?

we are likely going to have more updates in the future.

Reply:
of course